packages/alert/src/Alert.story.js

```diff
@@ -6,10 +6,12 @@ import { action } from "@storybook/addon-actions";
 import Alert from ".";
 import { POSITIVE, NEWS, NEGATIVE, ATTENTION } from "./constants/alertTypes";
 
-const ExtendedAlert = styled(Alert)`
+const StyledAlert = styled(Alert)`
   transform: rotate(5deg);
 `;
 
+const ExtendedAlert = props => <StyledAlert {...props} />;
```

Comment:
Do know why the previous version won't work? I wonder if this is the best approach to handle this, because the `styled(Alert)` should produce a valid component 🤔

Reply:
Ouch...